app/workers/virus_scan_worker.rb

```diff
@@ -9,11 +9,8 @@ def perform(asset_id)
     if scanner.clean?
       asset.scanned_clean
     else
-      Airbrake.notify_or_ignore(VirusScanner::InfectedFile.new, error_message: scanner.virus_info, params: { id: asset.id, filename: asset.filename })
+      GovukError.notify(VirusScanner::InfectedFile.new, extra: { error_message: scanner.virus_info, id: asset.id, filename: asset.filename })
       asset.scanned_infected
     end
-  rescue => e
-    Airbrake.notify_or_ignore(e, params: { id: asset.id, filename: asset.filename })
```

Comment:
The error in Sentry will just contain the `asset_id` (via the worker params). I'm assuming that we're okay with that.